x-pack/plugins/search_inference_endpoints/public/components/inference_endpoints_header.tsx

```diff
@@ -5,34 +5,33 @@
  * 2.0.
  */
 
-import { EuiButton, EuiPageTemplate } from '@elastic/eui';
+import { EuiPageTemplate, EuiLink } from '@elastic/eui';
 import React from 'react';
 import * as i18n from '../../common/translations';
+import { docLinks } from '../../common/doc_links';
+import { useTrainedModelPageUrl } from '../hooks/use_trained_model_page_url';
 
-interface InferenceEndpointsHeaderProps {
-  setIsInferenceFlyoutVisible: (isVisible: boolean) => void;
-}
+export const InferenceEndpointsHeader: React.FC = () => {
+  const trainedModelPageUrl = useTrainedModelPageUrl();
 
-export const InferenceEndpointsHeader: React.FC<InferenceEndpointsHeaderProps> = ({
-  setIsInferenceFlyoutVisible,
-}) => (
-  <EuiPageTemplate.Header
-    css={{ '.euiPageHeaderContent > .euiFlexGroup': { flexWrap: 'wrap' } }}
-    data-test-subj="allInferenceEndpointsPage"
-    pageTitle={i18n.INFERENCE_ENDPOINT_LABEL}
-    description={i18n.MANAGE_INFERENCE_ENDPOINTS_LABEL}
-    bottomBorder={true}
-    rightSideItems={[
-      <EuiButton
-        key="newInferenceEndpoint"
-        color="primary"
-        iconType="plusInCircle"
-        data-test-subj="addEndpointButtonForAllInferenceEndpoints"
-        fill
-        onClick={() => setIsInferenceFlyoutVisible(true)}
-      >
-        {i18n.ADD_ENDPOINT_LABEL}
-      </EuiButton>,
-    ]}
-  />
-);
+  return (
+    <EuiPageTemplate.Header
+      data-test-subj="allInferenceEndpointsPage"
+      pageTitle={i18n.INFERENCE_ENDPOINT_LABEL}
+      description={i18n.MANAGE_INFERENCE_ENDPOINTS_LABEL}
+      bottomBorder={true}
+      rightSideItems={[
+        <EuiLink href={trainedModelPageUrl} target="_blank" data-test-subj="view-your-models">
+          {i18n.VIEW_YOUR_MODELS_LINK}
+        </EuiLink>,
+        <EuiLink
+          href={docLinks.createInferenceEndpoint}
+          target="_blank"
+          data-test-subj="learn-more-about-inference-endpoints"
+        >
+          {i18n.LEARN_MORE_ABOUT_INFERENCE_ENDPOINTS_LINK}
+        </EuiLink>,
```

Comment:
Due to the length of these links. I think we can move the Inference endpoint link inside the description (accepts a React Node). 

@daveyholler @leemthompo Pinging you all for a second opinion.

```tsx
description={
<>
<EuiText>
<p>{i18n.MANAGE_INFERENCE_ENDPOINTS_LABEL}
<br/>
<EuiLink 
href={docLinks.createInferenceEndpoint}>{INFERENCE_ENDPOINTS_LINK}
target="_blank"
data-test-subj="learn-more-about-inference-endpoints"
>
{i18n.LEARN_MORE_ABOUT_INFERENCE_ENDPOINTS_LINK}
</EuiLink>
</p>
</EuiText>
</>
}
```

Reply:
I don't have any issues with that.

Reply:
Updated